common/types/shared.go

```diff
@@ -5565,13 +5769,20 @@ type SignalWithStartWorkflowExecutionRequest struct {
 	Control                             []byte                 `json:"control,omitempty"`
```

Comment:
may want to suppress `SignalInput` as well (one line above, but I can't comment there 😢)

Reply:
done